riak/datadog_checks/riak/data/conf.yaml.example

```diff
@@ -1,67 +1,90 @@
+#
 instances:
 
     ## @param url - string - required
     ## Riak stats url to connect to.
     #
   - url: http://127.0.0.1:8098/stats
 
-    ## @param min_collection_interval - integer - required - default: 60
-    ## Riak statistics are calculated on a per-minute sliding window,
-    ## Change the default collection interval if need be
-    #
-    min_collection_interval: 60
-
     ## @param cacert - string - optional
     ## Path to the TLS CA certificate file (ca.pem).
     #
     # cacert: <TLS_CA_CERT_PATH>
 
-    ## @param disable_cert_verify - boolean - optional - default: false
+    ## @param disable_cert_verify - boolean - optional - default: False
     ## Set to true to disable CA cert verification.
     #
     # disable_cert_verify: false
 
-    ## @param tags - list of key:value elements - optional
-    ## List of tags to attach to every metric, event, and service check emitted by this Integration.
+    ## @param tags - list of strings - optional
+    ## A list of tags to attach to every metric and service check emitted by this instance.
     ##
-    ## Learn more about tagging: https://docs.datadoghq.com/tagging/
+    ## Learn more about tagging at https://docs.datadoghq.com/tagging
     #
     # tags:
     #   - <KEY_1>:<VALUE_1>
     #   - <KEY_2>:<VALUE_2>
 
-## Log Section (Available for Agent >=6.0)
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Overrides any `service` defined in the `init_config` section.
+    #
+    # service: <SERVICE>
+
+    ## @param min_collection_interval - number - optional - default: 15
+    ## This changes the collection interval of the check. For more information, see:
+    ## https://docs.datadoghq.com/developers/write_agent_check/#collection-interval
+    #
+    # min_collection_interval: 15
+
+    ## @param empty_default_hostname - boolean - optional - default: false
+    ## This forces the check to send metrics with no hostname.
+    ##
+    ## This is useful for cluster-level checks.
+    #
+    # empty_default_hostname: false
+
+## Log Section
 ##
-## type - mandatory - Type of log input source (tcp / udp / file / windows_event)
-## port / path / channel_path - mandatory - Set port if type is tcp or udp. Set path if type is file. Set channel_path if type is windows_event
-## service - mandatory - Name of the service that generated the log
-## source  - mandatory - Attribute that defines which Integration sent the logs
-## tags: - optional - Add tags to the collected logs
+## type - required - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel_path - required - Set port if type is tcp or udp.
+##                                         Set path if type is file.
+##                                         Set channel_path if type is windows_event.
+## source  - required - Attribute that defines which Integration sent the logs.
```

Comment:
```suggestion
## source  - required - The attribute that defines the integration sending the logs.
```

Reply:
This should actually be done directly in the template log file and it will fix it for all the integrations https://github.com/DataDog/integrations-core/blob/master/datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/logs.yaml